oletools/olevba.py

```diff
@@ -1937,58 +1938,57 @@ def __init__(self, ole, vba_root, project_path, dir_path, relaxed=True):
+                referenceregistered_libid = unicode2str(self.decode_bytes(dir_stream.read(referenceregistered_sizeof_libid)))
+                log.debug(REFERENCEREGISTERED + ' lib id: %s' % referenceregistered_libid)
                 referenceregistered_reserved1 = struct.unpack("<L", dir_stream.read(4))[0]
-                self.check_value('REFERENCEREGISTERED_Reserved1', 0x0000, referenceregistered_reserved1)
+                self.check_value(REFERENCEREGISTERED + '_Reserved1', 0x0000, referenceregistered_reserved1)
                 referenceregistered_reserved2 = struct.unpack("<H", dir_stream.read(2))[0]
-                self.check_value('REFERENCEREGISTERED_Reserved2', 0x0000, referenceregistered_reserved2)
+                self.check_value(REFERENCEREGISTERED + '_Reserved2', 0x0000, referenceregistered_reserved2)
                 unused = referenceregistered_id
                 unused = referenceregistered_size
-                unused = referenceregistered_libid
+                self.references.append((REFERENCEREGISTERED,referenceregistered_libid))
                 continue
 
             if check == 0x000E:
-                # REFERENCEPROJECT
+                REFERENCEPROJECT = "REFERENCEPROJECT"
                 # Specifies a reference to an external VBA project.
                 referenceproject_id = check
                 referenceproject_size = struct.unpack("<L", dir_stream.read(4))[0]
                 referenceproject_sizeof_libidabsolute = struct.unpack("<L", dir_stream.read(4))[0]
-                referenceproject_libidabsolute = dir_stream.read(referenceproject_sizeof_libidabsolute)
-                log.debug('REFERENCE project lib id absolute: %s' % unicode2str(self.decode_bytes(referenceproject_libidabsolute)))
+                referenceproject_libidabsolute = unicode2str(self.decode_bytes(dir_stream.read(referenceproject_sizeof_libidabsolute)))
+                log.debug(REFERENCEPROJECT + ' lib id absolute: %s' % referenceproject_libidabsolute)
                 referenceproject_sizeof_libidrelative = struct.unpack("<L", dir_stream.read(4))[0]
-                referenceproject_libidrelative = dir_stream.read(referenceproject_sizeof_libidrelative)
-                log.debug('REFERENCE project lib id relative: %s' % unicode2str(self.decode_bytes(referenceproject_libidrelative)))
+                referenceproject_libidrelative = unicode2str(self.decode_bytes(dir_stream.read(referenceproject_sizeof_libidrelative)))
+                log.debug(REFERENCEPROJECT + ' lib id relative: %s' % referenceproject_libidrelative)
                 referenceproject_majorversion = struct.unpack("<L", dir_stream.read(4))[0]
                 referenceproject_minorversion = struct.unpack("<H", dir_stream.read(2))[0]
                 unused = referenceproject_id
                 unused = referenceproject_size
-                unused = referenceproject_libidabsolute
-                unused = referenceproject_libidrelative
```

Comment:
Actually, I think I need to leave the assignment unused = referenceproject_libidrelative because it isn't being used anywhere else.